tests/TestPC.py

```diff
@@ -330,3 +330,55 @@ def test_pc_load_bnlearn_discrete_datasets(self):
             print(f'{bname} ({num_nodes_in_truth} nodes/{num_edges_in_truth} edges): used {cg.PC_elapsed:.5f}s, SHD: {shd.get_shd()}')
 
         print('test_pc_load_bnlearn_discrete_datasets passed!\n')
+
+    # Test the usage of local cache checkpoint (check speed).
+    def test_pc_with_citest_local_checkpoint(self):
```

Comment:
I really like the tests here! 

Cheers! Now we can guarantee our PC algorithm is indeed written correctly. Great work!! :)